scripts/build

```diff
@@ -55,7 +77,7 @@ esac
 
 function rmStep(){
   rm -rf "$@"
-  rmdir -p --ignore-fail-on-non-empty `dirname "$@"`
+  #rmdir -p --ignore-fail-on-non-empty `dirname "$@"`
```

Comment:
> > +  #rmdir -p --ignore-fail-on-non-empty `dirname "$@"`
>
> Odd - Haven't had to use rmdir right after a rm -rf. Gives errors on OSX
related to the --ignore-fail-on-non-empty flag

It's just for clean-up and tidy-ness purposzez. What kind of errors are you
getting? This flag prevent to show error messages (and error status? Don't
remember...) when it get a not empty folder. Probably we could find a
workaround.